### DISCUSSION TITLE
ReferenceError: navigator is not defined when building vitepress project

### DISCUSSION THREAD

**Discussion by Amit091** [tinymce/tinymce#8436] [General]
What is the current behavior? Describe the bug.
 let's assume, we have a text editor equipped with  TinyMCE in vitepress, and as we build the project. Then we get the error `ReferenceError: navigator is not defined `

screenshot
![Screenshot](https://user-images.githubusercontent.com/25835996/217494987-28817d03-f798-4e99-82a5-12f0c2bda052.jpg)

TinyMCE version v6.3.1
vitepress version:  ^0.16.1
node version: 14.21.1

**Comment by lostkeys** [comment]
Hi @Amit091. We unfortunately cannot debug third party integrations. I will convert this issue into a discussion topic and maybe someone in the community have experience with Vitepress and can help out.
